src/schemas/v2/combined.json

```diff
@@ -36814,6 +36814,182 @@
               }
             }
           }
+        },
+        "valuationComparisonData": {
+          "type": "array",
+          "title": "Valuation Comparison Data",
+          "description": "A collection of data used to compare the valuations (market values) of properties in the real estate or property market. It typically includes various types of information about properties, such as recently sold properties, properties currently on sale, property pricing estimates, property characteristics, and more.",
+          "items": {
+            "type": "object",
+            "properties": {
+              "listingType": {
+                "title": "Listing Type",
+                "type": "string",
+                "enum": ["RecentlySold", "CurrentlyOnSale"]
+              },
+              "listingStatus": {
+                "title": "Listing Status ",
+                "type": "object",
+                "properties": {
+                  "type": {
+                    "type": "string",
+                    "enum": [
+                      "Sale",
+                      "Sold"
+                    ]
+                  },
+                  "date": {
+                    "title": "Date",
+                    "type": "string",
+                    "format": "date-time"
+                  },
+                  "price": {
+                    "title": "Price",
+                    "type": "object",
+                    "properties": {
+                      "amount": {
+                        "title": "Price",
+                        "type": "number"
+                      },
+                      "currency": {
+                        "title": "Currency",
+                        "type": "string",
+                        "enum": [
+                          "USD",
+                          "EUR",
+                          "GBP",
+                          "JPY",
+                          "CAD",
+                          "AUD",
+                          "Other"
+                        ]
+                      }
+                    }
+                  }
+                }
+              },
+              "propertyFullDescription": {
+                "title": "Property description",
+                "type": "string"
+              },
+              "uprn": {
+                "title": "Uprn",
+                "type": "string"
+              },
+              "propertyType": {
```

Comment:
in the main schema we don't constrain the property type to an enum, partly because for example the Rightmove ADF schema has about 100 options! 

If this needs to be an enum, I'd remove 'Property' (what does that mean?), and 'Others' should be 'Other' to match elsewhere in the schema, and should include the option to add what the 'Other' type is in the usual way.

Reply:
resolved